lib/system.nim

```diff
@@ -1146,7 +1146,7 @@ when defined(boehmgc):
   elif defined(macosx):
     const boehmLib = "libgc.dylib"
   elif defined(openbsd):
-    const boehmLib = "libgc.so.4.0"
+    const boehmLib = "libgc.so.(4|5).0"
```

Comment:
while this may work, it still sounds fragile when the v6 is introduced in future; could the following work, `const boehmLib = "libgc.so"` ? (assuming it's a symlink pointing to "latest") ?

Reply:
Unfortunately OpenBSD doesn't create such a symlink I don't believe.

Reply:
ok, LGTM; but maybe we should support `const boehmLib = "libgc.so.*.0"` ?